CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add DoesNotHaveLabels controller-runtime ListOption. This will check if a
+  lebel does not exist on an option when listing and deleting Objects.
```

Comment:
```suggestion
- Add DoesNotHaveLabels controller-runtime ListOption. This will check if a
  label does not exist on an option when listing and deleting Objects.
```